config/config.go

```diff
@@ -115,3 +116,23 @@ type Cert struct {
 	CertFile string `envconfig:"CERT_FILE"`
 	KeyFile  string `envconfig:"KEY_FILE"`
 }
+
+type Metrics struct {
+	Type     string
+	Log      LogMetrics
+	Syslog   SyslogMetrics
+	Interval time.Duration `default:"10s"`
+}
+
+type LogMetrics struct {
+	Out      string        `default:"stdout"`
+	Prefix   string        `default:"metrics"`
+	Flag     int           `default:"4"`
+	Interval time.Duration `default:"1m"`
+}
+
+type SyslogMetrics struct {
```

Comment:
__[golint]__
* [golint] exported type SyslogMetrics should have comment or be unexported

<!-- Comment by [SideCI](https://sideci.com) -->